frontend/util/formatters.tsx

```diff
@@ -12,3 +12,11 @@ export function formatPhone(phone) {
     return "";
   }
 }
+
+export function formatDateWithoutTime(timestamp) {
```

Comment:
What I don't like about this function is that it doesn't always return a value, which means it will implicitly return `undefined` sometimes.

Additionally, this function doesn't appear to be used anywhere. Let's delete it.